### PR TITLE
Updated to use system JWT for Queued Push to PHSA

### PR DESCRIPTION
**Pull request by bradhead**
## Status
**Ready**

## Fixes or Implements AB8450 NotificationSettings queued push to use system JWT
- [x] Enhancement
- [ ] Bug
- [ ] Documentation

## Description:
NotificationSettings queued push to use system JWT, but only when Queued, not direct.

## Testing
- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [x] Not Required

### Steps to Reproduce:
The push notificiation should authenticate with PHSA endpoint, not using user JWT,
which is not available to job scheduler, as either expired or not set. This way the Job Scheduler will obtain JWT at time to delivery to PHSA endpoint, allowing for validation.
the JSON notification structure adds subjectHdid as now required.

### UI Changes
NO

### Browsers Tested
- [ ] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments
We will need to update design notes to PHSA.

### CODE REVIEW

File: Apps/Common/src/Models/NotificationSettingsBase.cs [hunk 1]
```diff
@@ -38,6 +38,7 @@ public NotificationSettingsBase()
         /// <param name="notificationSettings">Initialize values from passed in object.</param>
         public NotificationSettingsBase(NotificationSettingsBase notificationSettings)
         {
+            this.SubjectHdid = notificationSettings.SubjectHdid;
```
Comment (sslaws) [inline Apps/Common/src/Models/NotificationSettingsBase.cs:41]: This needs to be moved to NotificationSettingsRequest as it is specific to the request object.  Unless you convince Tyler to echo it back :).

Comment (bradhead) [reply Apps/Common/src/Models/NotificationSettingsBase.cs:41]: we need hdid in the response to correlate or the response can't really be meaningfully processed in a stateless manner. I sent Tyler the change that we will be putting hdid in request and response payload. If there is a reason that's hardship we can move the hdid to the Request  only.

**Comment by bradhead** [comment]
I didn't touch a line of code in WebClient so why is it failing?

File: Apps/Common/src/Models/NotificationSettingsBase.cs (hunk 1 above)
Comment (bradhead) [reply Apps/Common/src/Models/NotificationSettingsBase.cs:41]: Can I get consensus on this? Would we ever need to process the response in the context of the hdid outside of the synchronous call? If not, I can put hdid only in request.  I did mention to Tyler we'd put it in both request and response.

**Pull request merged**